languages/js/src/Query.ts

```diff
@@ -323,6 +340,11 @@ export class Query {
               this.processMessages();
             });
             break;
+          }
+          default: {
+            const _: never = event.kind;
+            return _;
```

Comment:
is this is a typescript thing?

Reply:
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#exhaustiveness-checking